src/components/Events.jsx

```diff
@@ -1,31 +1,60 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
 import StyledEvent from './styledComponents/Sevents';
 
 function Events() {
+  const [actu, setActu] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(
+        'https://newsapi.org/v2/top-headlines?sources=techcrunch&apiKey=f58891b4575e4349939c7f3f38aff6ca'
+      )
+      .then(({ data }) => {
+        setActu(data.articles);
+      });
+  }, []);
+
   return (
     <StyledEvent>
-      <h1 className="eventsTitle">COMING EVENTS</h1>
-      <article className="Events">
-        <img
-          className="Event"
-          src="https://images.pexels.com/photos/577585/pexels-photo-577585.jpeg"
-          alt="Event"
-        />
-        <img
-          className="Event"
-          src="https://images.pexels.com/photos/577585/pexels-photo-577585.jpeg"
-          alt="Event"
-        />
-        <img
-          className="Event"
-          src="https://media.istockphoto.com/photos/application-programming-interface-connect-services-on-internet-and-picture-id1208498558"
-          alt="Event"
-        />
-        <img
-          className="Event"
-          src="https://media.istockphoto.com/photos/application-programming-interface-connect-services-on-internet-and-picture-id1208498558"
-          alt="Event"
-        />
-      </article>
+      <div className="InfoContainer">
+        <h1 className="InfoTitle">WHAT&apos;S NEW ?</h1>
+        <div className="Infos">
+          {actu
+            .sort(function compare(repoA, repoB) {
+              return repoB.publishedAt - repoA.publishedAt;
+            })
+            .slice(0, 2)
+            .map((subject) => {
+              return (
+                <div className="SubjectInfo">
+                  <img
+                    src={subject.urlToImage}
+                    className="SubjectImage"
+                    alt="SubjectImage"
+                  />
+                  <p className="SubjectTitle">
+                    <b>TITLE</b> : {subject.title}
+                  </p>
+                  <p className="SubjectDesc">
+                    <b>DESCRIPTION</b> : {subject.description}
+                  </p>
+                  <p className="SubjectUrl">
+                    <b>URL</b> :{' '}
+                    <a href={subject.url} target="_blank" rel="noreferrer">
+                      {subject.url}
+                    </a>
+                  </p>
+                  <p className="SubjectDate">
+                    <b>PUBLICATION DATE</b> :
```

Comment:
Stpo using "TITLE" (and other placeholders) : add enough style so that you user **knows** it's a title !